wiki/Installation/mac_installation/en.md

```diff
@@ -0,0 +1,41 @@
+# osu! titanic mac
+Hello, this guide is designed to help you if you are using Macintosh computer, run Titanic, a osu! server using a client of your choice, The project is developed by one person called Levi also known as Lekuru, and the person behind Titanic, to do this guide you'll need the following files:
+
+A client from the site
+The latest wineskin (the file will be in a zip on the GitHub.)
+A supported macintosh from the list below
+
+1. Make sure your Mac is running a operating system above High Sierra as it is required for the best performance and this guide, although it might run on High Sierra, most browsers don't support GitHub. **IMPORTANT! IF YOUR MAC IS RUNNING ANY CATALINA VERSION BELOW `10.15.5`, YOU'LL NEED TO DISABLE SYSTEM INTEGRITY PROTECTION (SIP)**
+
+Therefore, in order to run osu!mac on macOS Catalina `10.15.0` to `10.15.4` you need to disable SIP.
+
+To disable SIP, do the following:
+
+Restart your computer in Recovery mode.
+
+Turn on your Mac and immediately press and hold these two keys: Command (⌘) and `R`.
+
+Release the keys when you see an Apple logo, spinning globe or other startup screen.
+
+You may be prompted to enter a password, such as a firmware password or the password of a user who is an administrator of this Mac. Enter the requested password to continue.
+
+Launch Terminal from the Utilities menu. It should be in the menu bar at the top of your screen.
+
+Run the command csrutil disable.
+
+Restart your computer. You can do so from the top menu bar (just like you would normally.)
+
+Once you have SIP disabled, let it boot up normally, login, and you can then begin to install osu!.
+
+
+FOR THE BEST PERFORMANCE, PLEASE MAKE SURE YOUR MAC IS A 2011 OR HIGHER, 2008 or higher will run it but 2011 or higher as a recomendation.
+
+It also has to support 32/64 bit capabillites.
+
+1. Grab the whole package by clicking on Code and then download as ZIP, this will make it all into one big file, it is a big one so please be paitent on the download time as it can vary depending on your internet.
```

Comment:
```suggestion
1. Grab the whole package by clicking on "Code" and then "Download as ZIP". This will make it all into one big file. It is a big one, so please be patient on the download time as it can vary depending on your internet.
```